go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -73,6 +73,9 @@ func (tm *TabletManager) Backup(ctx context.Context, logger logutil.Logger, req
 	}
 	defer tm.endBackup(backupMode)
 
+	// create the loggers: tee to console and source
```

Comment:
Nit, but here and below we're not consistently using sentence capitalization and punctuation.